<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainGameWindow</class>
 <widget class="QMainWindow" name="MainGameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainGameWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Papyrus</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tic Tac Toe</string>
    </property>
   </widget>
   <widget class="QLabel" name="catLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(://images/smallCat.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dogLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(://images/smallDog.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="backgroundLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>501</width>
      <height>691</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(249, 235, 202);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="title2Label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Papyrus</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(X)Meow! vs Woof!(O)</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>530</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(166, 129, 84);background: rgb(255, 240, 229);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="exitPushButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>610</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="boardGroupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>381</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox { 
     border: 2px solid rgb(166, 129, 84); 
	background: rgb(255, 240, 229);
 } </string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="board01">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(125, 0, 188)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="board02">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(125, 0, 188)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="board00">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(125, 0, 188)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="board11">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>140</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(125, 0, 188)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="board10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(125, 0, 188)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="board12">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>140</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(125, 0, 188)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="board21">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>260</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(125, 0, 188)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="board20">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(125, 0, 188)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="board22">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>260</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(125, 0, 188)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="winnerCatLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>381</width>
       <height>381</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(://images/cat.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="winnerDogLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>381</width>
       <height>381</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(://images/dog.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="resetPushButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>610</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Reset!</string>
    </property>
   </widget>
   <zorder>backgroundLabel</zorder>
   <zorder>boardGroupBox</zorder>
   <zorder>titleLabel</zorder>
   <zorder>catLabel</zorder>
   <zorder>dogLabel</zorder>
   <zorder>title2Label</zorder>
   <zorder>statusLabel</zorder>
   <zorder>exitPushButton</zorder>
   <zorder>resetPushButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>487</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
